engine/combat/utility/sos_reasoner.py

```diff
@@ -15,7 +16,7 @@ def generate_considerations(
         considerations = []
         for player in players:
             if not player.dead and player.enabled:
-                considerations.append(Consideration(player))
+                considerations.append(SoSConsideration(player))
```

Comment:
As mentioned before, the missing function should be added to the bare `Consideration` class, even if just as a stub.

Reply:
i'll make this a ticket!